File: html/Lab9/3/1.js
//zad 1
let a = parseFloat(prompt("Podaj liczbę:"))
let b = parseFloat(prompt("Podaj liczbę:"))
let c = parseFloat(prompt("Podaj liczbę:"))

let p = document.getElementsByTagName("p");
let max = Math.max(a, b, c);
p[0].innerText += Math.max(a, b, c);
p[1].innerText += Math.min(a, b, c);
p[2].innerText += a + b + c - max;
p[3].innerText += Math.sqrt(a + b + c);

File: aisd/kolokwium/1.py
def maxpairdiff(l: list[int]) -> (int, int):
    m: (int, int) = None

    for i in range(0, len(l) - 1):
        if m is None:
            m = (l[i], l[i + 1])
        else:
            dm = max(m[0], m[1]) - min(m[0], m[1])
            dnew = max(l[i], l[i + 1]) - min(l[i], l[i + 1])
            if dnew > dm:
                m = (l[i], l[i + 1])

    return m


print(maxpairdiff([2, 5, 3, 7, 1, 4]))
print(maxpairdiff([2, 5, 0, 7, 1, 4]))
